docs/reference/indices/get-mapping.asciidoc

```diff
@@ -1,45 +1,90 @@
 [[indices-get-mapping]]
-== Get Mapping
+== Get mapping API
+++++
+<titleabbrev>Get mapping</titleabbrev>
+++++
 
-The get mapping API allows to retrieve mapping definitions for an index or
-index/type.
+Returns field mapping information for an index.
 
-[source,js]
---------------------------------------------------
-GET /twitter/_mapping
---------------------------------------------------
-// CONSOLE
-// TEST[setup:twitter]
+[float]
+[[indices-get-mapping-request]]
+=== {api-request-title}
 
-NOTE: Before 7.0.0, the 'mappings' definition used to include a type name. Although mappings
-in responses no longer contain a type name by default, you can still request the old format
-through the parameter include_type_name. For more details, please see <<removal-of-types>>.
+`GET /<index>/_mapping`
 
 [float]
-=== Multiple Indices
+[[indices-get-mapping-desc]]
+=== {api-description-title}
 
-The get mapping API can be used to get more than one index with a
-single call. General usage of the API follows the following syntax:
-`host:port/{index}/_mapping` where  `{index}` can accept a comma-separated
-list of names. To get mappings for all indices you can use `_all` for `{index}`.
-The following are some examples:
+You can use the get mapping API to retrieve field mapping information for one or
+more indices.
 
-[source,js]
---------------------------------------------------
-GET /twitter,kimchy/_mapping
---------------------------------------------------
-// CONSOLE
-// TEST[setup:twitter]
-// TEST[s/^/PUT kimchy\nPUT book\n/]
+[float]
+[[indices-get-mapping-path-params]]
+=== {api-path-parms-title}
 
-If you want to get mappings of all indices and types then the following
-two examples are equivalent:
+`<index>` (Optional)::
++
+--
+(string) Comma-separated list or wildcard expression of index names for which to
+retrieve field mapping information.
 
-[source,js]
---------------------------------------------------
-GET /_all/_mapping
+You can use a value of `_all` or `*` to retrieve mapping information for all
+indices.
+
+If no value is provided, the API retrieves mapping information for all indices.
+--
+
+[float]
+[[indices-get-mapping-query-params]]
+=== {api-query-parms-title}
+
+`allow_no_indices` (Optional)::
```

Comment:
I think there is some value in listing the parameters here rather than directing people to a separate page.

As a compromise, I moved these parameters to a separate file and included them. That way we can edit and reuse them from a single source but still make the content available everywhere.

Let me know if you think that works. I can also add an indicator that this is a rarely used or expert parameter if wanted.

Reply:
That seems like a good compromise to me, it will make the information easier to keep in sync.